src/du/du.rs

```diff
@@ -290,7 +295,9 @@ pub fn uumain(args: Vec<String>) -> i32 {
 
     let convert_size = |size: u64| -> String {
         if matches.opt_present("human-readable") || matches.opt_present("si") {
-            if size >= mb {
+            if size >= gb {
```

Comment:
Shouldn't this support TB, etc. as well?

Reply:
yeah it should, but then there will be 4 nearly duplicate lines not 3 :-(. I'll look at refactoring this bit.